### PR TITLE
 Portfolio project: Desktop Version

### PR DESCRIPTION
**Pull request by Hisoka37**
#  Set up Portfolio project: Desktop Version
In this PR I : 
- Add media queries to my portfolio website.
- Add CSS styling for the desktop screen. 
-  fix image declaration for better Github deployment.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,284 +1,309 @@
...
-        </div>
-      </section>
+      </div>
+    </section>
 
-      <section class="card">
+    <section class="card">
       <div class="grid-container">
         <div class="first-card">
           <img class="card-img" src="/assets/First Snapshoot Portfolio.svg" alt="Tonic">
```
Comment (pinkmoon25) [inline index.html]: - [x] Good job implementing the desktop version :clap: , I think you missed a `.` here due to which the image for this card is not rendering properly please see :point_down:  please change it to the correct path like :point_right:  `src=./assets/First Snapshoot Portfolio.svg`
image

File: styles.css [hunk 2]
```diff
@@ -427,3 +433,196 @@ h4 {
...
+
+  .card-img {
+    width: 54%;
+    display: flex;
+
+    /* justify-content: center; */
+  }
+
+  .descreption p {
+    font-size: 13px;
```
Comment (pinkmoon25) [inline styles.css]: - [x] Please see the font-size for the project card's paragraph is 16px in the original design :point_down: 
image

As one of the goal for the project is to parse a figma template a close to as possible, please fix this :pray:

File: styles.css [hunk 3]
```diff
@@ -427,3 +433,196 @@ h4 {
...
+    padding: 0;
+    height: 80px;
+    width: 372px;
+  }
+
+  .card-img {
+    width: 54%;
+    display: flex;
+
+    /* justify-content: center; */
```
Comment (pinkmoon25) [inline styles.css]: - [x] According to HTML CSS best practices we **Do not commit pieces of code as inline comments. They make your project look messy.** please remove all the commented code from the codebase :+1:

File: index.html [hunk 4]
```diff
@@ -1,284 +1,309 @@
...
-                <li  class="language-tag">html</li>
-                <li  class="language-tag">css</li>
-                <li  class="language-tag">Javascript</li>
+                <li class="language-tag">html</li>
+                <li class="language-tag">css</li>
+                <li class="language-tag">Javascript</li>
               </ul>
             </div>
-             <button class="see-project">See Project</button>
+            <button class="see-project">See Project</button>
```
Comment (pinkmoon25) [inline index.html:84]: - [x] As mentioned above one of the goal for this project is to parse a figma template as close as possible, how about implementing the button interactions to match the template design :point_down: 
image

- [x] Please also fix the border-raius property to match the original template :point_down: 
image

File: index.html [hunk 5]
```diff
@@ -1,284 +1,309 @@
...
-        </div>
-       
-        <div class="descreption" >
-          <p> A daily selection of privately
-              personalized reads; no accounts or 
-              sign-up required.</p>
-          </div>
+          <div class="my-project">
+
+            <h2 class="project-name">Tonic</h2>
```
Comment (pinkmoon25) [inline index.html:61]: - [x] Let's also fix the font-size for this heading as well, currently it is set to 32px, in the template however it is 40px :point_down: 
image

File: styles.css [hunk 6]
```diff
@@ -427,3 +433,196 @@ h4 {
...
+    padding-right: 43px;
+    color: #ff195e;
+  }
+
+  .desktop-menu-bar a {
+    color: #344563;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+  }
```
Comment (pinkmoon25) [inline styles.css]: - [x] Please notice there is no text-underline in the original design for the navbar links :point_down: 
image

let's fix this by adding a `text-decoration: none;`

File: styles.css [hunk 7]
```diff
@@ -427,3 +433,196 @@ h4 {
...
+    justify-content: space-around;
+    align-items: baseline;
+  }
+
+  .about-me-desktop {
+    width: 920px;
+    padding-left: 430px;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
```
Comment (pinkmoon25) [inline styles.css]: - [x] Please fix the font-size and line-height property for this para too to match the original design :point_down: 
image

File: index.html [hunk 8]
```diff
@@ -1,284 +1,309 @@
...
+            <input class="text" type="email" name="myEmail" placeholder="natajayanj14@gmail.com" required>
+          </div><br>
+          <div>
+            <textarea class="message" name="myText" maxlength="500" cols="30" rows="10"
+              placeholder="Write your message here" required></textarea>
+          </div>
+          <br>
+          <button type="submit" class="touch-button">Get in Touch</button>
+          <img class="under-line" src="./assets/underline.svg.svg" alt="">
+      </div>
```
Comment (pinkmoon25) [inline index.html:300]: - [x] I see we can Improve our contact form style to match the original design please see :point_down: 
image

what we have :point_down: 
image

1. We can reduce the margin between the form and the paragraph
2. fix the border-radius on our input and textarea element to be 2px as stated in the design template
3. fix the width on the contact paragraph to 569px as mentioned in the template

**Review by pinkmoon25 — CHANGES_REQUESTED** [review]
Hi @Hisoka37 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
linter checks :heavy_check_mark: 
used flex and grid :heavy_check_mark: 
follows figma design :heavy_check_mark: 
media query :heavy_check_mark: 
descriptive PR :heavy_check_mark: 
used github flow :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by binhussen — APPROVED** [review]
Hi @Hisoka37 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**